backend-comparison/src/persistence/base.rs

```diff
@@ -31,11 +31,11 @@ pub struct BenchmarkRecord {
 ///      "shapes": ["shape dimension", "shape dimension", ...],
 ///      "timestamp": "timestamp",
 ///      "numSamples": "number of samples",
-///      "min": "duration in seconds",
-///      "max": "duration in seconds",
-///      "median": "duration in seconds",
-///      "mean": "duration in seconds",
-///      "variance": "duration in seconds"
+///      "min": "duration in microseconds",
+///      "max": "duration in microseconds",
+///      "median": "duration in microseconds",
+///      "mean": "duration in microseconds",
+///      "variance": "duration in microseconds"
```

Comment:
I suppose this should be split into a separate commit. Ideally, it'd be great if this was in the benchmarking doc. It took me an attempt to change the formatting of results on stdout before I found out about the JSON files.